command/genesis/genesis.go

```diff
@@ -77,10 +78,17 @@ func setFlags(cmd *cobra.Command) {
 		"the maximum amount of gas used by all transactions in a block",
 	)
 
+	cmd.Flags().StringVar(
+		&params.burnContract,
+		burnContractFlag,
+		"",
+		"the burn contract block and address (format: <block>:<address>[:<burn destination>])",
+	)
+
 	cmd.Flags().StringVar(
 		&params.baseFeeConfig,
 		genesisBaseFeeConfigFlag,
-		"",
+		command.DefaultGenesisBaseFeeConfig,
```

Comment:
We still don't need this if london fork is disabled, so I'd leave it as an empty string by default. WDYT?